libass/ass.c

```diff
@@ -19,20 +19,12 @@
 #include "config.h"
 #include "ass_compat.h"
 
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <assert.h>
-#include <errno.h>
-#include <sys/types.h>
-#include <sys/stat.h>
 #include <inttypes.h>
 
 #ifdef CONFIG_ICONV
 #include <iconv.h>
 #endif
 
-#include "ass.h"
```

Comment:
I'm very confused. This file is literally defining the functions declared in this header. How is it inappropriate for it to include it?

Reply:
It's included through `ass_utils.h`. I don't think that header list has any impact on code clarity, so it's better for compiler to touch hard drive less.